hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/capacity/AbsoluteResourceCapacityCalculator.java

```diff
@@ -44,8 +44,14 @@ public double calculateMinimumResource(
     double remainingResourceRatio = resourceCalculationDriver.getRemainingRatioOfResource(
         label, resourceName);
 
-    return normalizedRatio * remainingResourceRatio * context.getCurrentMinimumCapacityEntry(
+    double minimumCapacity = context.getCurrentMinimumCapacityEntry(
```

Comment:
check: is this still needed?

Reply:
Looks like it's no longer needed, I'll run 2x test with both legacy and the new mode, without this change.